cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -27,124 +27,174 @@ class PredicateFunctionsTest extends DocumentingTest {
+        ("variable", "A variable that can be used from within the predicate."),
+        ("predicate", "A predicate that is tested against all items in the list."))
+      query("""MATCH p = (n)-[*1..3]->(b)
+              #WHERE
+              #  n.name = 'Alice'
+              #  AND none(x IN nodes(p) WHERE x.age = 25)
+              #RETURN p""".stripMargin('#'),
+      /*Result:
+      ({name: "Alice", eyes: "brown", age: 38})-[:KNOWS]->({name: "Charlie", eyes: "green", age: 53})
+      ({name: "Alice", eyes: "brown", age: 38})-[:KNOWS]->({name: "Charlie", eyes: "green", age: 53})-[:KNOWS]->({name: "Daniel", eyes: "brown", age: 54})
+      */
+      ResultAssertions(r => {
           r.toList.length should equal(2)
         })) {
-        p("No node in the returned paths has an `age` property set to *'25'*.")
+        p("No node in the returned paths has a property `age` with the value `25`.")
         resultTable()
       }
     }
     section("single()", "functions-single") {
-      p("`single()` returns true if the predicate holds for exactly one of the elements in the given list." +
-        " `null` is returned if the list is `null` or all of its elements are `null`.")
-      function("single(variable IN list WHERE predicate)", "A Boolean.", ("list", "An expression that returns a list."), ("variable", "This is the variable that can be used from within the predicate."), ("predicate", "A predicate that is tested against all items in the list."))
-      query(
-        """MATCH p = (n)-->(b)
-          |WHERE n.name = 'Alice'
-          |AND single(var IN nodes(p) WHERE var.eyes = 'blue')
-          |RETURN p""".stripMargin, ResultAssertions(r => {
+      p("""The function `single()` returns `true` if the predicate holds for exactly _one_ of the elements in the given list.
+          #`null` is returned if the list is `null` or all of its elements are `null`.""".stripMargin('#'))
+      function("single(variable IN list WHERE predicate)",
+        "A Boolean.",
+        ("list", "An expression that returns a list."),
+        ("variable", "A variable that can be used from within the predicate."),
+        ("predicate", "A predicate that is tested against all items in the list."))
+      query("""MATCH p = (n)-->(b)
+              #WHERE
+              #  n.name = 'Alice'
+              #  AND single(var IN nodes(p) WHERE var.eyes = 'blue')
+              #RETURN p""".stripMargin('#'),
+      /*Result:
+      ({name: "Alice", eyes: "brown", age: 38})-[:KNOWS]->({name: "Bob", eyes: "blue", age: 25})
+      */
+      ResultAssertions(r => {
           r.toList.length should equal(1)
         })) {
-        p("Exactly one node in every returned path has the `eyes` property set to *'blue'*.")
+        p("In every returned path there is exacly one node that has a property `eyes` with the value `'blue'`.")
```

Comment:
```suggestion
        p("In every returned path there is exactly one node that has a property `eyes` with the value `'blue'`.")
```

Reply:
I need to run a spell checker next time :)